misc/package/Data Files/MWSE/core/mcm/components/templates/Template.lua

```diff
@@ -331,34 +331,27 @@ function Template:register()
 end
 
 function Template.__index(tbl, key)
```

Comment:
The current implementation of `Template.__index` and `Category.__index` are basically piggybacking off of `mwse.mcm`. This may not be the most optimized approach, but it means that most of the complicated string logic is only done in `mwse/mcm.lua` and `mcm/fileUtils.lua`.

I'd be happy to change the implementations of `Template.__index` and `Category.__index` to something else if desired though.